internal/server/v1/alert/vars.go

```diff
@@ -0,0 +1,62 @@
+package alert
+
+import "golang.org/x/exp/maps"
+
+var SuppliedVariables = []string{"name", "team", "entity"}
```

Comment:
I am not sure with this but what are these supplied variables for?

Reply:
These are variables that are auto injected from dex. I'm not sure of their exact function as well honestly. This bit was done by @rohil .. in the refactor i retained the behavior and just moved things around.